### PR TITLE
Clinic

### PR DESCRIPTION
**Pull request by aylmss**
added mvc, services, initializer, links at header and at homepage. 

this is a test use case based on EmergencyReport example. additional columns will be added, need to check if correct for now

### CODE REVIEW

File: UtopiaCity/Data/Migrations/20210826122619_2.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace UtopiaCity.Data.Migrations
+{
+    public partial class _2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```
Comment (Ibraga96) [inline UtopiaCity/Data/Migrations/20210826122619_2.cs]: I think this migration should be removed

File: UtopiaCity/Common/DbInitializer.cs [hunk 2]
```diff
@@ -5,6 +5,7 @@
 using UtopiaCity.Common.Interfaces;
 using UtopiaCity.Data;
 
+
 namespace UtopiaCity.Common
```
Comment (crocomoth) [inline UtopiaCity/Common/DbInitializer.cs]: ?

File: UtopiaCityTest/UtopiaCityTest.csproj [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="5.0.8" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.15" />
```
Comment (crocomoth) [inline UtopiaCityTest/UtopiaCityTest.csproj]: do we need it?

**Review by crocomoth — CHANGES_REQUESTED** [review]

File: UtopiaCity/Common/DbInitializer.cs (hunk 2 above)
Comment (aylmss) [reply UtopiaCity/Common/DbInitializer.cs]: deleted, i assume it was a typo

File: UtopiaCityTest/UtopiaCityTest.csproj (hunk 3 above)
Comment (aylmss) [reply UtopiaCityTest/UtopiaCityTest.csproj]: deleted

**Review by Ibraga96 — APPROVED** [review]

**Review by crocomoth — APPROVED** [review]

**Pull request merged**